dydx/dydxPresenters/dydxPresenters/_v4/Vault/Landing/dydxVaultViewBuilder.swift

```diff
@@ -90,48 +91,61 @@ private class dydxVaultViewBuilderPresenter: HostedViewPresenter<dydxVaultViewMo
         viewModel?.thirtyDayReturnPercent = vault?.details?.thirtyDayReturnPercent?.doubleValue
         viewModel?.vaultBalance = vault?.account?.balanceUsdc?.doubleValue
         viewModel?.allTimeReturnUsdc = vault?.account?.allTimeReturnUsdc?.doubleValue.round(to: 2)
+        viewModel?.learnMoreAction = {
+            Router.shared?.navigate(to: RoutingRequest(path: "/vault/tos"), animated: true, completion: nil)
+        }
 
-        let newPositions = vault?.positions?.positions?.map { (position) -> dydxVaultPositionViewModel? in
-            guard let leverage = position.currentLeverageMultiple?.doubleValue,
-                  let marketId = position.marketId,
-                  // special case for fake USDC market to show unused margin
-                  let assetId = marketId == "USDC-USD" ? "USDC" : marketMap[marketId]?.assetId,
-                  let displayId = assetMap[assetId]?.id ?? marketMap[marketId]?.displayId
-            else { return nil }
-            let equity = position.marginUsdc?.doubleValue ?? 0
-            let notionalValue = position.currentPosition?.usdc?.doubleValue ?? 0
-            let positionSize = position.currentPosition?.asset?.doubleValue ?? 0
-            let iconType: PlatformIconViewModel.IconType
-            let tokenUnitPrecision: Int
-            if marketId == "USDC-USD" {
-                iconType = .asset(name: "symbol_USDC", bundle: .dydxView)
-                tokenUnitPrecision = 2
-            } else {
-                iconType = .init(url: URL(string: assetMap[assetId]?.resources?.imageUrl ?? ""), placeholderText: assetId.first?.uppercased())
-                tokenUnitPrecision = marketMap[marketId]?.configs?.displayStepSizeDecimals?.intValue ?? 2
+        let newPositions = vault?.positions?.positions?
+            .sorted { (lhs, rhs) -> Bool in
```

Comment:
is lhs/rhs left-hand-side and right-hand-side?

Reply:
Yeah